doc/locale/ja/LC_MESSAGES/news.po

```diff
@@ -153,10 +153,7 @@ msgid ""
 "[:doc:`/reference/functions/highlight_html`], [:doc:`reference/functions/"
 "snippet_html`] Improved that extract a keyword also from an expression of "
 "before executing a slices when we specify the slices parameter."
-msgstr ""
-"[:doc:`/reference/functions/highlight_html`], [:doc:`reference/functions/"
-"snippet_html`] slicesパラメータ指定時に、slices実行前の式からもキーワードを抽"
-"出するように改良しました。"
+msgstr "slicesパラメータ指定時に、slices実行前の式からもキーワードを抽出するように改良しました。"
```

Comment:
Why the following is removed? It's needed content.

```text
[:doc:`/reference/functions/highlight_html`], [:doc:`reference/functions/snippet_html`]
```

Reply:
I've edited wrong news entry. The next entry should be modified.